portafly/src/i18n/locales/en/accounts.yml

```diff
@@ -27,19 +27,25 @@ team_members_filter:
   by_name: Name
   by_email: Email
   by_sso_authentication: SSO authentication
-   yes: Yes
-   no: Not yet
+  by_sso_authentication_options:
+    yes: Yes
+    no: Not yet
+  by_sso_authentication_options_description: List of possible values of by_sso_authentication_options
```

Comment:
@porueesq I've added the descriptions

Reply:
@damianpm To avoid dependencies, we don't use variables in the descriptions. I've suggested the contents and descriptions here: https://github.com/3scale/porta/pull/1910/files#r430329124